### PR TITLE
[Guide] Completed recommended soundfonts section

### LINKED ISSUE #17: [Resource] Recommended Soundfonts
**What does this pertain to?**
This pertains to the OmniMIDI guides

**What is your suggestion?**
Finish adding recommended soundfonts to the OmniMIDI guides

**How will this help?**
This will help newcomers find good free soundfonts for Black MIDI to use

**Additional Context**
I will also implement video previews for each soundfont with about 4 MIDIs per

### PR DESCRIPTION
**Pull request by GreenWeegeeCantCode**
This one took quite a while to make, i put 5 amazing soundfonts for the section.
Also i just noticed that dependabot updated mkdocs material to 9.5.6, sorry if it breaks anything.

### CODE REVIEW

File: docs/OmniMIDI/Soundfonts/recommendations.md [hunk 1]
```diff
@@ -6,4 +6,39 @@ Here are some great soundfonts which we recommend you to use for you endevaours
 
     These soundfonts were created by amazing individuals who worked hard to create them and are kind enough to offer them for free for you to use. Please show some love by crediting the creators if you make videos or use them!
 
-> TODO: im not sure what soundfonts would be good. someone pls take over
+## Arachno
+
+Arachno is a really nice soundfont, it's one of the most used soundfonts in the Black MIDI Conmunity.
```
Comment (6gh) [inline docs/OmniMIDI/Soundfonts/recommendations.md]: Should we add, "for GM sounds" at the end of this? Most of the time arachno is used for its instruments other than the default piano one. I would also like to make it clear that this should be accompanied by another soundfont like the ones you added here.

File: docs/OmniMIDI/Soundfonts/recommendations.md [hunk 2]
```diff
@@ -6,4 +6,39 @@ Here are some great soundfonts which we recommend you to use for you endevaours
...
+It has a lot of instruments, such as drums, percussion and more!
+The download page for arachno can be found on the [Arachnosoft website.](https://www.arachnosoft.com/main/download.php?id=soundfont-sf2)
+
+Arachno is mainly a .sfARK file, however the download above is the .sf2 version which can be used in most synth's, including OmniMIDI
```
Comment (6gh) [inline docs/OmniMIDI/Soundfonts/recommendations.md:19]: The official download page isn't the best so if you could, may you also add a picture to show where the link is?

File: docs/requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-mkdocs-material==9.5.3
+mkdocs-material==9.5.6
```
Comment (6gh) [inline docs/requirements.txt]: This shouldn't be a problem, I was planning to merge the dependabot update anyways.

**Review by 6gh — COMMENTED** [review]
Nice job! Thank you for creating this. Right now I am on mobile so I will merge it later but I've left some feedback you might want to look over in the meantime.

File: docs/OmniMIDI/Soundfonts/recommendations.md (hunk 1 above)
Comment (GreenWeegeeCantCode) [reply docs/OmniMIDI/Soundfonts/recommendations.md]: Sure.

File: docs/OmniMIDI/Soundfonts/recommendations.md (hunk 2 above)
Comment (GreenWeegeeCantCode) [reply docs/OmniMIDI/Soundfonts/recommendations.md:19]: Ill add an image

**Review by 6gh — APPROVED** [review]
Page looks good! I will try to make preview videos for these soundfonts :+1:

Thank you for the contribution once again!

**Pull request merged**